src/pages/login/login.ts

```diff
@@ -57,25 +57,25 @@ export class LoginPage {
               this.contacts = contacts;
               console.log(contacts)
               this.goToContactList(contacts);
-            })
-          });
+            }).catch();
+          }).catch();
         }
-      })
-      .catch();
+      }).catch();
+
   }
 
   ionViewDidLoad() {
     console.log('ionViewDidLoad LoginPage');
   }
 
   goToInscription(){
-    this.navCtrl.push(InscriptionPage).then();
+    this.navCtrl.push(InscriptionPage).then().catch();;
```

Comment:
Really not sure that putting .then().catch() after a navCtrl request is important... Might make Webstorm happier but not in ionic guidelines

Reply:
double semicolon